src/components/legend/draw.js

```diff
@@ -742,6 +742,9 @@ function computeLegendDimensions(gd, groups, traces, opts) {
             opts._titleWidth + 2 * (bw + constants.titlePad)
         )
     );
+    if (!isVertical) {
```

Comment:
To pass the syntax test, 
no space required after `if` statement.
```js
if(!isVertical) {
```

Also you could run syntax test locally using:
```
npm run test-syntax
```